### ISSUE TITLE
ValueError - so close 

### ISSUE THREAD

**Issue by randmized** [GeorgeSeif/Transfer-Learning-Suite#8]
Hi George, 
after preparing my own dataset i run into the following error. 

https://i.imgur.com/1WUFlHa.png

Noting the numbers are very close, i tried different models and batch sizes but can not make progress. Do you have an idea what might cause the error?

This is the relevant line from main.py
` history = finetune_model.fit_generator(train_generator, epochs=args.num_epochs, workers=8, steps_per_epoch=num_train_images // BATCH_SIZE, 
        validation_data=validation_generator, validation_steps=num_val_images // BATCH_SIZE, class_weight='auto', shuffle=True, callbacks=callbacks_list)`

**Comment by randmized** [comment]
Clean your training folder before you start training. Sry George thank you for putting this tool together.
